src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant.Tests/GSoft.AspNetCore.Authentication.ClientCredentialsGrant.Tests.csproj

```diff
@@ -12,6 +12,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="FakeItEasy" Version="7.4.0" />
+    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="6.0.16" />
```

Comment:
Both packages aren't necessary.

Actually I think that we could only have a single test project. After all, we're already referencing both client & server projects in `GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests.csproj` (and it could be renamed to `GSoft.Authentication.ClientCredentialsGrant.Tests.csproj` in another story).